src/main/java/matrixConvertor.java

```diff
@@ -0,0 +1,34 @@
+public class matrixConvertor {
+    public static String convertor(String matrix, String word) {
```

Comment:
```suggestion
    public static String convertMatrix(String matrix, String word) {
```
для неймінгу методі краще використовувати дієслова

Reply:
done